srcpkgs/etlegacy/INSTALL

```diff
@@ -0,0 +1,14 @@
+# INSTALL
+case "${ACTION}" in
+	post)
+		if [ "${UPDATE}" = "no" ]; then
+			echo ""
+			echo "Downloading 219MB essential asset files for game to run."
+			for i in 0 1 2; do
+				xbps-fetch -o /usr/lib/etlegacy/etmain/pak$i.pk3 https://mirror.etlegacy.com/etmain/pak$i.pk3
```

Comment:
Again, doing that install time is bad idea. If package is restricted, this can be done buildtime with `distfiles`, `skip_extraction` and `vinstall`.

Reply:
@ Chocimier  Thank you, nice to learn new things